django/db/models/fields/__init__.py

```diff
@@ -92,7 +93,7 @@ def _empty(of_cls):
 
 @total_ordering
 @python_2_unicode_compatible
-class Field(RegisterLookupMixin):
+class Field(RegisterLookupMixin, FieldCacheMixin):
```

Comment:
Hmm, I just thought of this, but do we really need all Fields to inherit `FieldCacheMixin`? It's really only used by related fields, right? I mean, for normal local fields no cache is needed, the main field attribute on the model is effectively the "cache". Could we get away with having only related fields inherit the mixin?